README.md

```diff
@@ -80,4 +89,12 @@ Configuration files are placed in the ```startup/challenge/custom_config``` fold
 
 ## Challenge
 
-More details on the challenge can be found in the competition rulebook http://www.uasconferences.com/2022_icuas/uav-competition-rulebook-and-faq/
\ No newline at end of file
+More details on the challenge can be found in the competition rulebook http://www.uasconferences.com/2022_icuas/uav-competition-rulebook-and-faq/
+0
+| ![challenge.png](.fig/challenge.png) | 
+|:--:| 
+| UAV setup for the ICUAS 2022 challenge. |
+
+* ```challenge_started``` - After ```True``` is published on this topic the challenge is setup correctly and you can safely run your code.
+* ```spawn_ball``` - You can manually call this service at any point to reset the ball at the UAV.
+* ```uav_magnet/gain``` - When you want to detach the ball from the magnetic gripper, publish a ```0``` gain on this topic.
```

Comment:
Does spawn_ball service reset magnet gain or do we need to manually reset magnet gain before respawning the ball?

Reply:
@lmark1

Reply:
I added a note at the end of the challenge segment about the ball spawner service functionality in [d38dd51](https://github.com/larics/icuas22_competition/pull/3/commits/d38dd51e22f03b100328958abcece30544e34cb9).